docs/attributes-registry/log.md

```diff
@@ -40,12 +40,68 @@ Attributes for a file to which log was emitted.
 
 This document defines the generic attributes that may be used in any Log Record.
 
-| Attribute             | Type   | Description                                 | Examples                                                                                                                         | Stability                                                        |
-| --------------------- | ------ | ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
-| `log.record.original` | string | The complete orignal Log Record. [1]        | `77 <86>1 2015-08-06T21:58:59.694Z 192.168.2.133 inactive - - - Something happened`; `[INFO] 8/3/24 12:34:56 Something happened` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `log.record.uid`      | string | A unique identifier for the Log Record. [2] | `01ARZ3NDEKTSV4RRFFQ69G5FAV`                                                                                                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-
-**[1]:** This value MAY be added when processing a Log Record which was originally transmitted as a string or equivalent data type AND the Body field of the Log Record does not contain the same value. (e.g. a syslog or a log record read from a file.)
-
-**[2]:** If an id is provided, other log records with the same id will be considered duplicates and can be removed safely. This means, that two distinguishable log records MUST have different values.
-The id MAY be an [Universally Unique Lexicographically Sortable Identifier (ULID)](https://github.com/ulid/spec), but other identifiers (e.g. UUID) may be used as needed.
+| Attribute             | Type   | Description                                   | Examples                                                                                                                         | Stability                                                        |
+| --------------------- | ------ | --------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `log.record.id`       | string | A durable identifier for the Log Record. [1]  | `1`; `0x100F`                                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
```suggestion
| `log.record.id`       | int| A durable identifier for the Log Record. [1]  | `1`; `0x100F`                                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Unless there is a strong reason, consider making the durable id as an integer number, to make fast filtering etc. in hot path.